### ISSUE TITLE
GPU memory required to train r100

### ISSUE THREAD

**Issue by JingyuanHu** [TreB1eN/InsightFace_Pytorch#5]
Thank you for your great  work! However, I met a problem about limitation of GPU memory. Can you give me some suggestion about the minimum GPU memory required to train r100 ArcFace by your code?

**Comment by TreB1eN** [comment]
you mean resnet100? you can easily change the batchsize in the configuration if you face a memory issue, I hava 100 batchsize in resnet100 training

**Comment by JingyuanHu** [comment]
Thank you for your reply. Yes, I mean resnet100.  My GPU is one Titan xp. I can change batchsize to 64 when backbone is resnet50. I want to try resnet100 and larger batchsize for better results.  So I want to know the minimum GPU memory required.  Maybe I can change a  better to GPU?

**Comment by TreB1eN** [comment]
简单一点，如果你想要用大一点的batchsize,就直接把设置改了跑起来，然后用nvidia-smi 命令查看显存占用，多试几次就知道最大能放多少batchsize了，我一般是不会去算一个batch要占用多少显存的。提醒你batchsize不是越大越好的，而且改大了batchsize, learning rate也要调大。

**Comment by cvtower** [comment]
> 简单一点，如果你想要用大一点的batchsize,就直接把设置改了跑起来，然后用nvidia-smi 命令查看显存占用，多试几次就知道最大能放多少batchsize了，我一般是不会去算一个batch要占用多少显存的。提醒你batchsize不是越大越好的，而且改大了batchsize, learning rate也要调大。


So what is the corresponding paras for the accuracy data mentioned in this repo?
init lr:e-2 or e-3?   paper: e-1
batchsize: deponds(default value is 200 for e-3?)  paper:512
epoch: default value:8
Do you have any suggestion for 1080ti platform?
